### PR TITLE
adding PHP WF name

### PR DESCRIPTION
**Pull request by rachfop**
## What does this PR do?

Adds the PHP WF type/name to the docs

### CODE REVIEW

**Review by bergundy — APPROVED** [review]
I'll let @wolfy-j take this one, I just blindly approved.

**Review by wolfy-j — APPROVED** [review]

File: docs-src/php/how-to-customize-workflow-type-in-php.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+id: how-to-customize-workflow-type-in-php
+title: How to customize Workflow types in PHP
```
Comment (flossypurse) [inline docs-src/php/how-to-customize-workflow-type-in-php.md:3]:
```suggestion
title: How to customize a Workflow Type in PHP
```

File: docs-src/php/how-to-customize-workflow-type-in-php.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+sidebar_label: Customize Workflow types
```
Comment (flossypurse) [inline docs-src/php/how-to-customize-workflow-type-in-php.md:4]:
```suggestion
sidebar_label: Custom Workflow Type
```

File: docs-src/php/how-to-customize-workflow-type-in-php.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+description: Customize Workflow types.
```
Comment (flossypurse) [inline docs-src/php/how-to-customize-workflow-type-in-php.md:5]:
```suggestion
description: How to customize a Workflow Type in PHP
```

Comment (djmagee) [inline docs-src/php/how-to-customize-workflow-type-in-php.md:5]:
```suggestion
description: To customize a Workflow type, use the Workflow method annotations to specify the name of Workflow.
```

File: docs-src/php/how-to-customize-workflow-type-in-php.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+tags:
+  - developer-guide
+  - sdk
+  - php
+---
+
+To customize a Workflow type, use the Workflow method annotations to specify the name of Workflow.
```
Comment (flossypurse) [inline docs-src/php/how-to-customize-workflow-type-in-php.md:12]:
```suggestion
To customize a Workflow Type, use the `WorkflowMethod` annotation to specify the name of Workflow.
```

File: docs-src/php/how-to-customize-workflow-type-in-php.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+```php
+#[WorkflowMethod(name)]
+```
+
+If a Workflow type is not specified, then Workflow type defaults to the interface name, `FileProcessingWorkflow`.
```
Comment (djmagee) [inline docs-src/php/how-to-customize-workflow-type-in-php.md:18]:
```suggestion
If not specified, Workflow type defaults to the interface name, `FileProcessingWorkflow`.
```

Comment (flossypurse) [inline docs-src/php/how-to-customize-workflow-type-in-php.md:18]:
```suggestion
If a Workflow Type is not specified, then Workflow Type defaults to the interface name, which is `FileProcessingWorkflow` in this case.
```

File: docs-src/php/how-to-customize-workflow-type-in-php.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+```php
+#[WorkflowInterface]
+interface FileProcessingWorkflow
```
Comment (flossypurse) [inline docs-src/php/how-to-customize-workflow-type-in-php.md:22]: We should make this YourWorkflowDefinitionInterface

**Review by flossypurse — APPROVED** [review]

**Review by flossypurse — APPROVED** [review]

**Pull request merged**